### DISCUSSION TITLE
Your graphics drivers do not support OpenGL 2.0.

### DISCUSSION THREAD

**Discussion by RobinsonCrosoe** [3b1b/manim#1631] [Q&A] [answered]
I am getting this error while opening GuiTest from example_scenes.py please help!
```
Your graphics drivers do not support OpenGL 2.0.
You may experience rendering issues or crashes.
Microsoft Corporation
GDI Generic
1.1.0
```

**Comment by TonyCrane** [comment, marked as answer]
Update your graphics drivers.
And I think you are using the community version of manim. So you'd better ask this question on their discord channel, or open an issue in ManimCommunity/manim instead of here.
